charts/opentelemetry-operator/values.yaml

```
@@ -220,6 +220,9 @@ admissionWebhooks:
   ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
   autoGenerateCert: true
 
+  # Set true to reuse existing cert, autoGenerateCert must be enabled too
+  autoGenerateCertIfNotExists: true
```

The following configuration for reference only. If you apply this comments, you may need to change the logic in `operator-webhook.yaml`

```suggestion
  ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
  autoGenerateCert: 
    enabled: true
    # use the old secret  auto generated before, applicable to helm upgrade case.
    reuseExisting: true
```

or 

```suggestion
  ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
  autoGenerateCert: 
    enabled: true
    # true means always re-generate cert
    always: true
```